src/Microsoft.AspNet.Authentication.Google/GoogleHandler.cs

```diff
@@ -31,12 +31,8 @@ public GoogleHandler(HttpClient httpClient)
             response.EnsureSuccessStatusCode();
 
             var payload = JObject.Parse(await response.Content.ReadAsStringAsync());
-            
-            var context = new OAuthCreatingTicketContext(Context, Options, Backchannel, tokens, payload)
-            {
-                Properties = properties,
-                Principal = new ClaimsPrincipal(identity)
-            };
+
+            var context = new OAuthCreatingTicketContext(new ClaimsPrincipal(identity), properties, Context, Options, Backchannel, tokens, payload);
```

Comment:
Yeah its seems big, but I would assume all of these are required for OAuth ticket contexts?  Making them properties doesn't really change anything except moving them into a { } block and making it possible to forget to set one...